src/lib/stitching/exchange/__tests__/stitching.test.ts

```diff
@@ -28,6 +28,18 @@ it("extends the Order objects", async () => {
   }
 })
 
+it("resolves amount fields on CommerceOrder", async () => {
```

Comment:
@ashkan18 added a test for the resolver specifically-- this would fail before because `resolvers.CommerceOrder` didn't exist, but I think it's still good 👍 .